Chargebee/Classes/ReceiptValidation/CBReceiptValidationManager.swift

```diff
@@ -31,6 +31,9 @@ struct CBReceipt {
     let customerId: String?
     let period:Int
     let periodUnit:Int
+    let first_name: String?
+    let last_name: String?
+    let email: String?
```

Comment:
Also would it make sense to use the `CBCustomer` object here inside?

Reply:
yes updated as CBCustomer !

Reply:
Can `CBReceipt` be composed of `CBCustomer` object instead of separate fields?

Reply:
I have updated  CBReceipt